src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,65 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
move to `*` selector and delete here and https://github.com/mate-academy/layout_product-cards/pull/3530/files#diff-74df4d86d0cb91ed11f5325dec8149ff23f54c0d521ff31c1a15bdabffa3e435R3